ui/app/rif/pages/lumino/tokensPage/tokensPage.js

```diff
@@ -0,0 +1,102 @@
+import React, {Component} from 'react'
+import {connect} from 'react-redux'
+import {pageNames} from '../../index';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import rifActions from '../../../actions';
+import PropTypes from 'prop-types';
+import { tokenIcons, brandConnections, DEFAULT_ICON } from '../../../constants/icons';
+import {Menu} from '../../../components';
+
+const PATH_TO_RIF_IMAGES = '/images/rif/';
```

Comment:
You should put this in the constants file

Reply:
There i moved it to the constants file